clients/javascript/test/game/api_spec.js

```diff
@@ -1,6 +1,4 @@
 const chai = require('chai');
-const chaiAsPromised = require('chai-as-promised');
-chai.use(chaiAsPromised);
 const expect = chai.expect;
 const sinon = require('sinon');
```

Comment:
[Sinon.js](http://sinonjs.org/) is a mocking framework. I use it to create fake objects to be able to have more specific tests.